cryptoconditions/condition.py

```diff
@@ -74,63 +94,98 @@ def from_uri(serialized_condition):
         Returns:
             Condition: Resulting object
         """
+        # TODO consider removing.
         if isinstance(serialized_condition, Condition):
             return serialized_condition
+        # TODO Use static typing instead (e.g.: with mypy).
         elif not isinstance(serialized_condition, str):
             raise TypeError('Serialized condition must be a string')
 
         pieces = serialized_condition.split(':')
-        if not pieces[0] == 'cc':
-            raise ValueError('Serialized condition must start with "cc:"')
-
-        if not re.match(CONDITION_REGEX_STRICT, serialized_condition):
-            raise ValueError('Invalid condition format')
-
+        if pieces[0] != CONDITION_URI_SCHEME:
```

Comment:
Is there a need for this check here? Wouldn't the `regex_match` also catch this error?